### PR TITLE
test(e2e): [simpleauth/frontend/function/bot] can deploy to customized rg

### PR DESCRIPTION
**Pull request by dilin-MS**
1. https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12898867
2. https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12898873
3. https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12898874
4. https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12898875

### CODE REVIEW

**Review by KennethBWSong — DISMISSED** [review]
Approve for simple auth

**Review by chagong — APPROVED** [review]

**Review by Alive-Fish — DISMISSED** [review]

**Review by YitongFeng-git — APPROVED** [review]
lgtm

**Review by LongOddCode — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Review by chagong — DISMISSED** [review]

**Review by YitongFeng-git — DISMISSED** [review]

File: packages/cli/tests/e2e/bot/Arm_Bot_CanDeployToCustomizedRg.tests.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      `name: 'addTeamsFxBotConfiguration'`
+    );
+
+    // Provision
+    setBotSkuNameToB1Bicep(projectPath, environmentManager.getDefaultEnvName());
+    await CliHelper.setSubscription(subscription, projectPath);
+    await CliHelper.provisionProject(projectPath);
+
+    // deploy
+    await CliHelper.deployProject(ResourceToDeploy.FrontendHosting, projectPath);
```
Comment (IvanJobs) [inline packages/cli/tests/e2e/bot/Arm_Bot_CanDeployToCustomizedRg.tests.ts:69]: Ask a question to clarify my confusion - In this case, provision bot but deploy frontend?

Comment (dilin-MS) [reply packages/cli/tests/e2e/bot/Arm_Bot_CanDeployToCustomizedRg.tests.ts]: fixed. Here we need to deploy bot. And this cli [bug](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12905612) hides this bug without throwing out error. @Alive-Fish will help track the bug.

**Review by YitongFeng-git — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Review by LongOddCode — APPROVED** [review]

**Review by KennethBWSong — APPROVED** [review]

**Review by adashen — APPROVED** [review]

**Review by IvanJobs — APPROVED** [review]
lgtm for tab/bot/function

**Pull request merged**